new command hardis:project:generate:bypass

Mehdi-Cloudity opened the pull request.

nvuillam reviewed (dismissed):
Well played :)

nvuillam commented:
@Mehdi-Cloudity you didn't test 😋 

![image](https://github.com/user-attachments/assets/4ab8303f-9110-4579-a6f4-6cd9f6cf4105)

You need to add the parameters on the command on sfdx-hardis :) (just copy paste them from another)

And you can also select an existing icon or add one -> https://github.com/hardisgroupcom/vscode-sfdx-hardis/blob/3cdceac1104e3a374a7309102fada8cfaa4d5b26/src/themeUtils.ts#L152

VsCode icons catalog is here -> https://code.visualstudio.com/api/references/icons-in-labels

Mehdi-Cloudity commented:
Added necessary flags on sfdx-hardis : https://github.com/hardisgroupcom/sfdx-hardis/pull/1069 and added an icon + tested 😋 


![image](https://github.com/user-attachments/assets/d5df82a2-395f-4ad4-95f7-7533094bdea7)
